src/main/java/aquality/selenium/core/configurations/ElementCacheConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package aquality.selenium.core.configurations;
+
+import aquality.selenium.core.utilities.ISettingsFile;
+import com.google.inject.Inject;
+
+public class ElementCacheConfiguration implements IElementCacheConfiguration{
+
+    private static final String JSON_PATH = "/elementCache/isEnabled";
```

Comment:
could you please rename as IS_ENABLED_PATH

Reply:
fixed